src/index.ts

```diff
@@ -254,9 +268,11 @@ export class Pool extends (EventEmitter as new () => PoolEmitter) {
 
     const defaultOptions: PoolOptionsBase = {
       poolSize: 10,
+      minPoolSize: 0,
       idleTimeoutMillis: 10000,
       waitForAvailableConnectionTimeoutMillis: 90000,
-      connectionTimeoutMillis: 30000,
+      connectionTimeoutMillis: options.minPoolSize && options.minPoolSize > 0 ? 0 : 30000,
+      query_timeout: 30000,
```

Comment:
By default, the pg library [leaves this unspecified](https://github.com/brianc/node-postgres/blob/4b229275cfe41ca17b7d69bd39f91ada0068a5d0/packages/pg/lib/defaults.js#L62) and I'm thinking we should do the same.